realm-core/lib/realm/event_router.rb

```diff
@@ -63,7 +63,7 @@ def init_gateways(gateways_spec)
     end
 
     def gateway_class(type)
-      return InternalLoopGateway if type == :internal_loop
+      return InternalLoopGateway if type.to_s == 'internal_loop'
```

Comment:
@davidsevcik Why you change to string?? Symbols are more preferable.

Reply:
Because when it comes from ENV it's a string and I don't want to rely on converting it in app code. It could be as well `type.to_sym == :internal_loop`. Just in back of my mind I remember that converting symbols to string was faster than other way around, but maybe it's not true anymore... Do you know?

Reply:
@davidsevcik Unfortunately I don't remember but it's ok. I asked the question only for my curiosity ;-)